semapv-properties.tsv

```diff
@@ -0,0 +1,14 @@
+IRI	skos:prefLabel	skos:definition	dc:source	skos:example	rdfs:comment	altLabel	Parent
```

Comment:
add column `rdfs:isDefinedBy`

Reply:
Intended to hold what type of value? An IRI to the defining standard (e.g. SKOS)? In that case should that replace `dc:source`?

Reply:
Its the standard property we use to say: this term is defined in FBBT. This term is defined in Mondo. This term is defined in SKOS. Its intended to reference the actual RDF, not a website